src/ansys/dpf/core/operators/metadata/coordinate_system_data_provider.py

```diff
@@ -0,0 +1,323 @@
+"""
+coordinate_system_data_provider
+
+Autogenerated DPF operator classes.
+"""
+
+from warnings import warn
+from ansys.dpf.core.dpf_operator import Operator
+from ansys.dpf.core.inputs import Input, _Inputs
+from ansys.dpf.core.outputs import Output, _Outputs
+from ansys.dpf.core.operators.specification import PinSpecification, Specification
+
+
+class coordinate_system_data_provider(Operator):
+    """Reads coordinate systems data from the result files contained in the
+    streams or data sources.
+
+    Parameters
+    ----------
+    solver_coordinate_system_ids : int, optional
+        Coorfinate system ids to recover used by the
```

Comment:
```suggestion
        Coordinate system ids to recover used by the
```
@oparreno can we really ask for several? The typehint says it expects a single `int`.